stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.18.0
+version: 1.18.1
```

Comment:
Since we are adding a feature lets bump the minor version to `1.19.0`

Reply:
I was thinking it might need a major version bump since resources are being renamed like the controller PSP and role. Not sure if that's considered a breaking change or not.

Reply:
will do